pkg/proxy/iptables/proxier.go

```diff
@@ -854,30 +856,38 @@ func (proxier *Proxier) syncProxyRules() {
 				writeLine(proxier.natChains, utiliptables.MakeChainLine(svcXlbChain))
 			}
 			activeNATChains[svcXlbChain] = true
-		} else if activeNATChains[svcXlbChain] {
```

Comment:
What happened to this?

Reply:
AFAICT, there's no way `activeNATChains[svcXlbChain]` could be already set at this point, since `activeNATChains` starts out empty, no other service would have the same `svcXlbChain` name, and no other part of the loop modifies `activeNATChains[svcXlbChain]`. It looks like cruft from an earlier version of the code.